Assets/Scripts/Poached/Audio/SoundManager.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using Poached.Utility;
+using UnityEngine;
+
+public class SoundManager : SingletonMonoBehaviour<SoundManager>
```

Comment:
Refer to PauseManager and how it's created on GameManager

Reply:
FIxed